includes/polling/mempools/gwd.inc.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2019 hartred <tumanov@asarta.ru>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+d_echo('gwd: ');
+$perc = snmp_get($device, '1.3.6.1.4.1.10072.2.20.1.1.2.1.1.20.1.1', '-OvQ', 'GW-EPON-MIB');
+if (is_numeric($perc)) {
+    $mempool['perc'] = $perc;
+    $mempool['used'] = $perc;
+    $mempool['total'] = 100;
+    $mempool['free'] = 100 - $perc;
```

Comment:
total, used and free should be "bytes" and not percents. You need to do the conversion depending on the data provided by the device.

Reply:
done